stestr/commands/run.py

```diff
@@ -42,8 +43,10 @@ def set_cli_opts(parser):
     parser.add_argument("--load-list", default=None,
                         help="Only run tests listed in the named file."),
     parser.add_argument("--partial", action="store_true", default=False,
-                        help="Only some tests will be run. Implied by "
-                             "--failing.")
+                        help="DEPRECATED: Only some tests will be run. Implied"
+                             " by --failing. This option is deprecated and no "
+                             "longer does anything. It will be removed in the"
+                             "future")
```

Comment:
Do you want to add a white space before `future`?

Reply:
oh, good catch. I'll respin this in a bit